edx_proctoring/templates/emails/proctoring_attempt_status_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+Dear {{ student_name }},
+
+    This email is to notify that status of {{ exam_name }} for {{ course_name }} has changed to {{ status }} on {{ date }}.
+
+Thank you.
+
+
```

Comment:
Thanks for including i18n at the top, but these strings are not wrapped as translatable. See some of the Django templates in edx_proctoring/templates/... to see the pattern that is used. 

Also, @catong @griffresch we are going to need some real text here. Are you available to provide that? There are three states: pending review, satisfactory review, failed review, which I presume need different text.

@hasnain-naveed depending on the complexity of the differences, we might actually need three different email templates to make it more legible (from a code perspective) rather than having lots of 'if-conditionals' in the template.